devices/Paexo/src/Paexo.cpp

```diff
@@ -346,10 +418,48 @@ bool Paexo::open(yarp::os::Searchable& config)
                                                                                                                                         pImpl->ftSensorPrefix + "Right" + pImpl->ftSensorName)};
 #endif
 
-    // Initialize wearable actuators
+    // Initialize wearable actuators for left and right motor
     pImpl->motorActuatorPrefix = getWearableName() + actuator::IMotor::getPrefix();
-    pImpl->paexoMotorActuator = ElementPtr<PaexoImpl::PaexoMotorActuator>{std::make_shared<PaexoImpl::PaexoMotorActuator>(pImpl.get(),
-                                                                                                                         pImpl->motorActuatorPrefix + pImpl->motorActuatorName)};
+    const std::string leftActuatorName = pImpl->motorActuatorPrefix + "Left" + pImpl->motorActuatorName;
+    pImpl->paexoLeftMotorActuator = ElementPtr<PaexoImpl::PaexoMotorActuator>{std::make_shared<PaexoImpl::PaexoMotorActuator>(pImpl.get(), leftActuatorName)};
+
+    const std::string rightActuatorName = pImpl->motorActuatorPrefix + "Right" + pImpl->motorActuatorName;
+    pImpl->paexoRightMotorActuator = ElementPtr<PaexoImpl::PaexoMotorActuator>{std::make_shared<PaexoImpl::PaexoMotorActuator>(pImpl.get(), rightActuatorName)};
+
+    // Initialize yarp control ports
+
+    // Check yarp network initialization
+    if (!yarp::os::Network::isNetworkInitialized()) {
+        yInfo() << LogPrefix << "Initializing yarp network";
+        yarp::os::Network();
+    }
```

Comment:
The typical patter we use to initialize the network is adding a `Network` object in the pImpl. When constructed, it automatically initializes the yarp network and, when it's destructed, it finalizes it.

Reply:
Done in https://github.com/robotology/wearables/pull/124/commits/a7f49d2dbb42898b22ceca1ea7a3b3f536dc3222